Sources_v3/StreamChat/Database/DTOs/MessageDTO.swift

```diff
@@ -281,7 +289,9 @@ extension _ChatMessage {
         showReplyInChannel = dto.showReplyInChannel
         replyCount = Int(dto.replyCount)
         isSilent = dto.isSilent
-        reactionScores = dto.reactionScores
+        reactionScores = dto.reactionScores.reduce(into: [:]) {
+            $0[.init(rawValue: $1.key)] = $1.value
+        }
```

Comment:
This is repeated 3 times, we might need a helper. Also I didn't totally understand why we need to `reduce` it

Reply:
Yep, we need something like `mapValues` but for keys -> `mapKey` :)

Reply:
`mapKeys` is implemented in 24cc3daf34d39da8aede5d8a21d13a2cf5e120a9